reduction/lr_reduction/event_reduction.py

```diff
@@ -615,6 +620,8 @@ def specular(self, q_summing=False, tof_weighted=False, bck_in_q=False, clean=Fa
         # Compute Q resolution
         self.dq_over_q = compute_resolution(self._ws_sc, theta=self.theta, q_summing=q_summing)
         self.q_summing = q_summing
+        self.wavelength, self.d_lambda = compute_wavelength_resolution(self._ws_sc)
```

Comment:
move this line right after the line with the call to `compute_resolution()`, and make a  `TODO` code comment stating that this component of the Q-resolution will be added into `self.dq_over_q`